Source/Scene/ImageryProvider.js

```diff
@@ -296,7 +296,7 @@ ImageryProvider.prototype.getTileCredits = function (x, y, level) {
  * @param {Number} y The tile Y coordinate.
  * @param {Number} level The tile level.
  * @param {Request} [request] The request object. Intended for internal use only.
- * @returns {Promise.<HTMLImageElement|HTMLCanvasElement>|undefined} A promise for the image that will resolve when the image is available, or
+ * @returns {HTMLImageElement|HTMLCanvasElement|ImageBitmap|Promise.<HTMLImageElement|HTMLCanvasElement|ImageBitmap>|undefined} A promise for the image that will resolve when the image is available, or
```

Comment:
Per #10265 , is this an undesired change?  Or would it be better if the caller had to expect to handle both immediate value-returns as well as a returned Promise?  (If y'all switched over to async/await this would be a lot simpler...)

Reply:
Also the issue would have been much more visible if you used actual ES6 classes with inheritance rather than having "ImageryProviders" that don't actually inherit from ImageryProvider, not sure if that's still on the table....

Reply:
I think we want to have each `ImageryProvider` implementation return a `Promise` or `undefined`. Yes, I agree it's difficult to enforce with the current state of things, but full migration to ES6 classes is a bit out of scope for this PR. 😅